bundles/org.openhab.binding.anel/src/main/java/org/openhab/binding/anel/internal/AnelHandler.java

```diff
@@ -0,0 +1,380 @@
+                        + config;
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, msg);
+            }
+
+            try {
+                refreshRequestWithoutResponse++;
+
+                udpConnector2.send(IAnelConstants.BROADCAST_DISCOVERY_MSG);
+                sendingFailures = 0;
+
+            } catch (Exception e) {
+
+                handleSendException(e);
+            }
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        final AnelUdpConnector udpConnector2 = udpConnector;
+        if (udpConnector2 == null || !udpConnector2.isConnected() || getThing().getStatus() != ThingStatus.ONLINE) {
+
+            // don't log initial refresh commands because they may occur before thing is online
+            if (!(command instanceof RefreshType)) {
+                logger.debug("Cannot handle command '{}' for channel '{}' because thing ({}) is not connected: {}", //
+                        command, channelUID.getId(), getThing().getStatus(), config);
+            }
+            return;
+        }
+
+        String anelCommand = null;
+        if (command instanceof RefreshType) {
+
+            final State update = stateUpdater.getChannelUpdate(channelUID.getId(), state);
+            if (update != null) {
+
+                updateState(channelUID, update);
+
+            } else if (!refreshRequested) {
+
+                // send broadcast request for refreshing the state; remember it to avoid multiple simultaneous requests
+                refreshRequested = true;
+                anelCommand = IAnelConstants.BROADCAST_DISCOVERY_MSG;
+
+            } else {
+
+                logger.debug(
+                        "Channel {} received command {} which is ignored because another channel already requested the same command",
+                        channelUID, command);
+            }
+        } else if (command instanceof OnOffType) {
+
+            final State lockedState;
+            synchronized (this) { // lock needed to update the state if needed
+
+                lockedState = commandHandler.getLockedState(state, channelUID.getId());
+                if (lockedState == null) {
+
+                    // command only possible if state is not locked
+                    anelCommand = commandHandler.toAnelCommandAndUnsetState(state, channelUID.getId(), command,
+                            getAuthentication());
+                }
+            }
+
+            if (lockedState != null) {
+                logger.debug("Channel {} received command {} but it is locked, so the state is reset to {}.",
+                        channelUID, command, lockedState);
+
+                updateState(channelUID, lockedState);
+
+            } else if (anelCommand == null) {
+                logger.info("Channel {} received command {} which is (currently) not supported", channelUID, command);
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be debug. See https://next.openhab.org/docs/developer/guidelines.html#f-logging

Please check all.

Reply:
I thought that this would be valuable information for the user because he/she configured a rule / sitemap / whatever to send a command to the channel that is not supported. With debug loglevel, this information does not reach the user by default.

How else do you want to report such cases to the user?

Reply:
You could make it warn.

Reply:
Done. Now all your findings should be resolved.